src/engine/renderer/tr_shader.cpp

```diff
@@ -1845,22 +1845,22 @@ static bool ParseStage( shaderStage_t *stage, const char **text )
 				ParseSpecularMap( stage, text );
 			}
 		}
-		else if ( !Q_stricmp( token, "materialMap" ) )
+		else if ( !Q_stricmp( token, "physicalMap" ) )
```

Comment:
This is compatibility breaking, right? Is it the case that zero maps are using this?

Reply:
There is zero map using it. That's something we created and never used yet.

And if we discover another engine had the same bad idea (and I've checked some of them and found no one), we can re-add this keyword as a _compatibility layer for third party content_.